pkg/controller/command/verifiable/models.go

```diff
@@ -55,8 +53,8 @@ type ProofOptions struct {
 	SignatureType string `json:"signatureType,omitempty"`
 	// PrivateKey is used to sign instead of DID
 	PrivateKey string `json:"privateKey,omitempty"`
-	// DIDKeyID id of the public key associated with the private key
-	DIDKeyID string `json:"didKeyID,omitempty"`
+	// proofPurpose is purpose of the proof.
+	proofPurpose string
```

Comment:
we might need to revisit if all scenarios always have the same `authentication` purpose in the VP.

Reply:
I am already validating if privatekey & verification methods are matching DID authentication purpose.

Reply:
right - i meant we might need to revisit the assumption that `purpose` is always `authentication.`

Reply:
i.e., `authentication` seems like a good default but the question to be revisited: is it always `authentication`?

Reply:
created https://github.com/hyperledger/aries-framework-go/issues/1750